code/game/gamemodes/sandbox/sandbox.dm

```diff
@@ -6,10 +6,15 @@
 
 	announce_span = "info"
 	announce_text = "Build your own station... or just shoot each other!"
-	
+
 	allow_persistence_save = FALSE
 
 /datum/game_mode/sandbox/pre_setup()
+	//Disables Random Events
+	var/new_are = !CONFIG_GET(flag/allow_random_events)
+	CONFIG_SET(flag/allow_random_events, new_are)
+
+	//Enables player spawing
```

Comment:
please surround your core code changes with
`//austation begin -- <reason for change>`
and
`//austation end`

Reply:
also you should probably just force it to false, instead of inverting the config value.